paccmann_chemistry/models/training.py

```diff
@@ -142,6 +146,11 @@ def train_vae(
         )
         loss.backward()
         train_loss += loss.detach().item()
+        # for name, param in vae_model.named_parameters():
+        #     if param.grad is None:
+        #         print(name)
+        #         print(param.grad)
```

Comment:
Should this commented out section be removed?

Reply:
thx, removed it.